common/src/vote/submission.rs

```diff
@@ -387,6 +403,26 @@ mod tests {
         let json1 = serde_json::to_string(&dummy_submissions).unwrap();
         let json2 = serde_json::to_string(&submissions).unwrap();
         assert_eq!(json1, json2);
+
+        let dummy_submissions = DummySubmissions {
+            total_submission_weight: TokenAmount::from_atto(100),
+            most_voted_key: vec![],
+            submitters: Default::default(),
+            submission_weights: Default::default(),
+            submissions: Default::default(),
+        };
+
+        let submissions = EpochVoteSubmissions::<DummyVote> {
+            total_submission_weight: TokenAmount::from_atto(100),
+            most_voted_key: None,
+            submitters: Default::default(),
+            submission_weights: Default::default(),
+            submissions: Default::default(),
+        };
+
+        let json1 = serde_json::to_string(&dummy_submissions).unwrap();
```

Comment:
Also, ideally this tests should check the cbor serialization (which is the one that we used). Theoretically they should be the same, but bugs may happen :shrug: